imsv-docs-docusaurus/openapi/endpoints/file-uploads/upload-file.yaml

```diff
@@ -0,0 +1,46 @@
+tags:
+  - file-uploads
+summary: Upload A File
```

Comment:
This is a regression: https://www.notion.so/immersve/Remove-a-from-endpoint-titles-83c254c7704e4d80a4a6c68772bd3887?pvs=4